internal/controllers/clusterextension_controller.go

```diff
@@ -212,6 +212,9 @@ func checkForUnexpectedFieldChange(a, b ocv1alpha1.ClusterExtension) bool {
 */
 //nolint:unparam
 func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alpha1.ClusterExtension) (ctrl.Result, error) {
+	l := log.FromContext(ctx)
```

Comment:
Did you want to put the name of the `ClusterExtension` here?

Reply:
Possibly, also attach the V(1) to this. e.g.
```suggestion
	l := log.FromContext(ctx).V(1).AddValues("ClusterExtension", ext.GetName())
```

Reply:
The name is already part of the context provided in the context that arrives from the caller of `Reconcile`.

I thought about using `V(1)`, but that would mean that no logging (with that logger at least) could ever happen at the INFO level.

Reply:
I was going to suggest that I could do something like this:
```
l := log.FromContext(ctx)
debugLog := l.V(1)
```

But that would be misleading because levels build (i.e. `.V(1).V(1)` is equivalent to `.V(2)`), so if the caller set a `.V(1)` logger in the context, the this would suddenly _not_ be a debug logger.